protostar/testing/starkware/pure_cairo_test_execution_state.py

```diff
@@ -0,0 +1,15 @@
+from dataclasses import dataclass
+
+from protostar.testing.stopwatch import Stopwatch
+from protostar.testing.test_output_recorder import OutputRecorder
+
+
+@dataclass
+class PureCairoTestExecutionState:
```

Comment:
I would name it `CairoTestExecutionState` to be consistent with `CairoCompiler`

Reply:
done